docs/high-availability/etcd.manifest

```diff
@@ -0,0 +1,97 @@
+{
+"apiVersion": "v1beta3",
```

Comment:
Hmmm... This whole wave is for v1beta3 in correspondence with the containers and the original ha doc creation .
 How about we merge it as is and then later a separate commit that updates the recipe for v1 ?